Add Gujarati as new language for React doc

harshilparmar opened the pull request:
Sorry , I don't have have direct experience to contribute to translation yet but I had started contributing to old react dev doc: https://github.com/stale-reactjs-org-translations/gu.reactjs.org/issues/1 
https://github.com/stale-reactjs-org-translations/gu.reactjs.org/issues/6

But other than translation, I also made contribution to redux doc: https://github.com/reduxjs/react-redux/pull/1655 
And I always tries to give my contribution to react:
https://github.com/reactjs/react.dev/issues/3012  
https://twitter.com/Harshil__parmar/status/1578469780092485633?s=20 
 
I hope these might help to evaluate me.

gaearon reviewed (changes requested):
As a part of fixing the permission sync process, we've changed how we track languages — please see https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9. 
Could you please update this PR:

- Rebase on top of latest `main`, and add the language to `lang/langs.json` instead
- Include the list of maintainers into the PR comment body instead

Thank you, and sorry for the churn.

harshilparmar commented:
> As a part of fixing the permission sync process, we've changed how we track languages — please see [63a10c4](https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9). Could you please update this PR:
> 
> * Rebase on top of latest `main`, and add the language to `lang/langs.json` instead
> * Include the list of maintainers into the PR comment body instead
> 
> Thank you, and sorry for the churn.

Thanks @gaearon , I have updated this PR.

idarshan181 commented:
Please add me as a maintainer, I would like to maintain and contribute to translating it to Gujarati actively.
